docs/getting-started/user-guides/testnet-voting.md

```diff
@@ -0,0 +1,183 @@
+
+> Step 5: Purchase Testnet Tickets
+
+Click the "Purchase tickets" tab within the Paymetheus navigation menu. You will see 7 form fields within the page. All of the defaults can be used for purchasing tickets **except** the "Stake pool preference". Click the "Manage pools" button. You need to enter the API key for your account at the testnet stakepool. To do this, simply visit [https://teststakepool.decred.org/settings](https://teststakepool.decred.org/settings) - your API Token should be the first item on the page. Enter it into the API Key field within Paymetheus and press Save. Your 1-of-2 multi-sig script will be automatically generated and you can press Close. From here, select teststakepool.decred.org from the Stake pool preference drop down and press the Purchase button begin purchasing tickets! Note: the ticket difficulty is equal to the cost per ticket, so make sure you have enough testnet coins to purchase at least one.
+
+> Step 6: Set The Votebits of Your Tickets via Stakepool 
+
+When you buy tickets via a stakepool, you automatically give that pool control of the votebits. The pool can then vote whichever way it pleases. (Note, if you purchase tickets without a stakepool, your ticket votebits default to 1)
+
+*Alternatively* (and recommended for this guide), a user can set the votebits via the stakepool tickets interface. Below is a screenshot of the [https://teststakepool.decred.org/tickets](https://teststakepool.decred.org/tickets) page. At the bottom of the "Live/Immature" section of this page, you will see the votebit settings. You can only edit the votebits of *all* of your tickets at once via the pool's interface. The tickets displayed below were set to "Yes" for "Previous Block Valid?" and "Yes" for "Increase Block Size from 1.0 MiB to 1.25MB" which resulted in a Votebit value of 5. 
+
+<img src="/img/testnet-voting_votebit-setting.jpg">
+
+For some basic information about Votebits, visit [An Explanation of Votebits](#an-explanation-of-votebits).
+
+## Command Line Instructions
+
+> Step 1: Download and Install Decred
+
+If you haven't already updated your Decred binaries to v0.8.2, visit the [Installation Guide](/getting-started/install-guide.md) and follow the directions for your operating system.
+
+> Step 2: Create Config Files
+
+If you already familiar with the `.conf` files, continue to step 3.
+
+All of the Decred software, when started, reads from a configuration file to determine which settings it should enable/disable/set during that initial load. All of the command line startup flags `(e.g. dcrwallet --testnet)` can be replaced by settings within the appropriate configuration file `(e.g. dcrwallet --testnet could be replaced by testnet=1 in dcrwallet.conf)`.
+
+These configuration files are located with the data directory of the application. The default data directory for Windows, macOS/OSX, and Linux are listed below:
```

Comment:
Those really are not the default data directories.  The default directories on linux for example is
```
~/.dcrd/ and ~/.dcrwallet/
```
with similar ones for the other oses.

Also, really should not call that the data directory since we already have a data directory ( -b, --datadir=             Directory to store data (/home/jcv/.dcrd/data).  The directory you are referring to is the application home directory:
 -A, --appdata=             Path to application home directory
                             (/home/jcv/.dcrd)